llvm/lib/Target/AArch64/AArch64InstrInfo.td

```diff
@@ -2145,6 +2145,12 @@ def MSR_FPSR : Pseudo<(outs), (ins GPR64:$val),
                PseudoInstExpansion<(MSR 0xda21, GPR64:$val)>,
                Sched<[WriteSys]>;
 
+let Uses = [FPMR], Defs = [FPMR, NZCV] in
+def SET_FPMR : Pseudo<(outs), (ins GPR64:$val),
```

Comment:
I think it best to match the existing naming scheme, which would make this `def MSR_FPMR`.